### ISSUE TITLE
WeMos Lolin D1 Mini V4.0 compatibility

### ISSUE THREAD

**Issue by sym0nd0** [OpenSourceDistilling/The_Jeffrey_PCB#1]
Hi,

Firstly, thanks for all the work done on this project.

I've been trying, and failing, to build an iSpindel using the Jeffrey v2.69.

Unfortunately, it seems like all of the D1 Mini V3 Pros I've bought, from different sources, have all been fake and refuse to work. From [further research](https://github.com/universam1/iSpindel/issues/560#issuecomment-998256815), it appears that the V3 has now gone EoL and the [V4 is the current version](https://www.wemos.cc/en/latest/d1/d1_mini.html).

The V4 has a [470Ω resistor (R14) between RST and D0 (IO16)](https://imgur.com/a/Sp4ZyKL), the pin used to wake the board from deep sleep. There's a [known issue](https://github.com/universam1/iSpindel/issues/59) where, once the battery voltage drops, with a 470Ω resistor in use, the board can't be woken up from deep sleep.

Lolin have confirmed which resistor is [R14](https://imgur.com/a/q1wWNQy).

I've some V4s and I'm thinking that I'll be able to remove the [470Ω resistor](https://imgur.com/a/q1wWNQy), bridge the pads, and the BAT43 diode, as part of the Jeffrey PCB build, will work perfectly and allow for wake from deep sleep.

The [schematics for the V4](https://www.wemos.cc/en/latest/_static/files/sch_d1_mini_v4.0.0.pdf) also appear to show that there's no longer a diode on the board.

So, as I ramble on, can anyone who knows what they're talking about (as I don't) please advise if I'm correct in assuming that there is no longer a diode on the board?

Thanks for any help!

TL;DR Can the D1 Mini V4 be used, if the 470Ω resistor between RST and D0 is removed? And is there no longer a diode on the board?

**Comment by PharmaJap** [comment]
To my eye, The Jeffery creates its own path from RST to D0/IO16, through the BAT43 diode. That should mean that as long as JP1 remains **open**/unbridged, R14 is part of an unused open circuit. So there should be no need to remove R14 or bridge JP1.

My v4 builds are waking properly so far with an unmodified board; currently running battery drain tests to confirm.

I believe you're correct about the VBUS diode being removed. Besides the schematic, it certainly looks like the trace runs directly to/under the USB-C socket, presumably to A4/B9. I haven't yet removed the socket from a board to see what's going on under there, and what the trace on the obverse connects to.

Honestly, I just stuck a silicone dust cover in the port, as a reminder not to use it without taking safety measures first. You can program the D1 Mini before assembly (safest), with the battery removed and the switch off, or just with the switch off (least safe).

**Comment by sym0nd0** [comment]
> To my eye, The Jeffery creates its own path from RST to D0/IO16, through the BAT43 diode. That should mean that as long as JP1 remains **open**/unbridged, R14 is part of an unused open circuit. So there should be no need to remove R14 or bridge JP1.
> 
> My v4 builds are waking properly so far with an unmodified board; currently running battery drain tests to confirm.
> 
> I believe you're correct about the VBUS diode being removed. Besides the schematic, it certainly looks like the trace runs directly to/under the USB-C socket, presumably to A4/B9. I haven't yet removed the socket from a board to see what's going on under there, and what the trace on the obverse connects to.
> 
> Honestly, I just stuck a silicone dust cover in the port, as a reminder not to use it without taking safety measures first. You can program the D1 Mini before assembly (safest), with the battery removed and the switch off, or just with the switch off (least safe).

Absolute superstar! Thank you.

I'll give it a whirl with a V4 and Jeffrey V2.69 tomorrow.

**Comment by sym0nd0** [comment]
Closed as project abandoned.
